packages/protocol/contracts-0.8/common/CeloUnreleasedTreasure.sol

```diff
@@ -49,8 +47,7 @@ contract CeloUnreleasedTreasure is UsingRegistry, ReentrancyGuard, Initializable
    */
   function release(address to, uint256 amount) external onlyEpochManager {
     require(address(this).balance >= amount, "Insufficient balance.");
-    IERC20 celoToken = IERC20(address(getCeloToken()));
-    celoToken.transfer(to, amount);
+    require(getCeloToken().transfer(to, amount), "CELO transfer failed.");
```

Comment:
I think you can do this with SafeTransfer library

Reply:
figured it was cleaner to do it this way, to avoid importing a library just for this transfer. The additional check of the library is not realy helpfule to us, since we already know the call is made to a contract